bundles/org.openhab.binding.nibeuplink/src/main/resources/ESH-INF/thing/readonly-channel-types.xml

```diff
@@ -0,0 +1,122 @@
+
+	<channel-type id="type-number-unscaled">
+		<item-type>Number</item-type>
+		<label>Unnamed Number</label>
+		<state pattern="%d" readOnly="true">
+		</state>
+	</channel-type>
+	<channel-type id="type-number-scale10">
+		<item-type>Number</item-type>
+		<label>Unnamed Number (0.1)</label>
+		<state pattern="%.1f" readOnly="true">
+		</state>
+	</channel-type>
+	<channel-type id="type-number-scale100">
+		<item-type>Number</item-type>
+		<label>Unnamed Number (0.01)</label>
+		<state pattern="%.2f" readOnly="true">
+		</state>
+	</channel-type>
+	<channel-type id="type-switch">
+		<item-type>Switch</item-type>
+		<label>Unnamed Switch</label>
+		<state readOnly="true"></state>
+	</channel-type>
+	<channel-type id="type-temperature">
+		<item-type>Number:Temperature</item-type>
+		<label>Unnamed Temperature</label>
+		<state pattern="%.1f %unit%" readOnly="true">
+		</state>
+	</channel-type>
+	<channel-type id="type-flow">
+		<item-type>Number:Dimensionless</item-type>
+		<label>Unnamed Flow</label>
+		<state pattern="%.1f l/min" readOnly="true">
+		</state>
+	</channel-type>
+	<channel-type id="type-electric-current">
+		<item-type>Number:ElectricCurrent</item-type>
+		<label>Unnamed Current</label>
+		<state pattern="%.1f %unit%" readOnly="true">
+		</state>
+	</channel-type>
+	<channel-type id="type-time-unscaled">
+		<item-type>Number:Time</item-type>
+		<label>Unnamed Time</label>
+		<state pattern="%d %unit%" readOnly="true">
+		</state>
+	</channel-type>
+	<channel-type id="type-time-scale10">
+		<item-type>Number:Time</item-type>
+		<label>Unnamed Time</label>
+		<state pattern="%.1f %unit%" readOnly="true">
+		</state>
+	</channel-type>
+	<channel-type id="type-frequency-unscaled">
+		<item-type>Number:Frequency</item-type>
+		<label>Unnamed Frequency</label>
+		<state pattern="%d %unit%" readOnly="true">
+		</state>
+	</channel-type>
+	<channel-type id="type-frequency-scale10">
+		<item-type>Number:Frequency</item-type>
+		<label>Unnamed Frequency</label>
+		<state pattern="%d %unit%" readOnly="true">
+		</state>
+	</channel-type>
+	<channel-type id="type-power">
+		<item-type>Number:Power</item-type>
+		<label>Unnamed Power</label>
+		<state pattern="%.2f %unit%" readOnly="true">
+		</state>
+	</channel-type>
+	<channel-type id="type-energy">
+		<item-type>Number:Energy</item-type>
+		<label>Unnamed Energy</label>
+		<state pattern="%.1f %unit%" readOnly="true">
+		</state>
+	</channel-type>
+	<channel-type id="type-pressure">
+		<item-type>Number:Pressure</item-type>
+		<label>Unnamed Pressure</label>
+		<state pattern="%.1f %unit%" readOnly="true">
+		</state>
+	</channel-type>
+	<channel-type id="type-speed-percent">
+		<item-type>Number:Dimensionless</item-type>
+		<label>Unnamed Speed</label>
+		<state pattern="%d %%" readOnly="true">
+		</state>
+	</channel-type>
+	<channel-type id="type-defrosting-state">
+		<item-type>Number</item-type>
```

Comment:
This is just copied from the old implementation which was working fine.
Using a number type instead of String has the benefit that this Channel can be persisted usinf InfluxDB/Grafana which does not support String type channels.

So, as this is working for a long time, I would prefer to keep it as it is.